supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
+* Users first: Allows users to navigate intuitively.
+* Allows Google to understand architecture of the site.
+* Establishes a quantifiable pathway to recognize topical expertise.
+* They increase site visitation time.
+* Helps Google crawl your site.
+* Builds relevancy and ranking signals for keywords and topics.
+* Builds citation or helps users add context.
+
+.Technical best practices
+* Use standard hyperlinks:
+`<a href="http://www.same-domain.com/" title="Keyword Text">Keyword Text</a>`
+* Do not use non hyperlinks.
```

Comment:
I'm not sure about this one. Are you still talking only about internal linking?

If linking to third party sites, then including a URL instead of a hyperlink is a good way to avoid broken links. In fact, it's best to provide a high-level URL with a non-link title (e.g. For more info, see _Installing a vGPU_ at www.nvidia.com).  At my previous workplace, this was our stated guidance (_not_ to use hyperlinks to third-party sites) True, when the link works it's less useful, but when the link doesn't work, it's a lot more useful and less annoying.

Reply:
I like this guidance about third-party sites. Maybe we should include it explicitly.